### ISSUE TITLE
Enable rule "import/order"

### ISSUE THREAD

**Issue by LinusU** [standard/standard#1490]
I think it would be nice to add the `import/order` rule. This is my suggestion for how the rule should be configured:

```json
{
  "alphabetize": {
    "order": "asc"
  },
  "groups": [
    ["builtin", "external"],
    ["internal", "unknown"],
    ["parent"],
    ["sibling", "index"]
  ],
  "newlines-between": "always"
}
```

This is based on what I've seen mostly when contributing in mostly the Node.js ecosystem. I'm very open to hearing other peoples perspective!

I think that a good strategy could be to start with only the `groups` option, and then follow up with the other options later.

Just a quick rundown on what it enforces:

```js
import bodyParser from 'body-parser'
import express from 'express'

import projects from './api/projects'
import users from './api/users'
```

```js
import http from 'http'

import bar from '../lib/bar'
import foo from '../lib/foo'

import info from './'
```

The benefit of this rule:

- Cleaner git diffs and less merge conflicts since there is a predictable place for each import
- I know that many people prefers having external dependencies first, and thus modify the file manually after using auto-imports in their IDE, this is now automatic
- You can easily see which external dependencies a specific file has
- It looks neat and clean

This rule is auto-fixable! 🎉

**Comment by mightyiam** [comment]
@LinusU how do you feel about #1517 ?

**Comment by LinusU** [comment]
@mightyiam that specific plugin doesn't support `require`, that is probably going to be quite a big problem for us? 🤔

**Comment by mightyiam** [comment]
Yes. Only where they are mixed. Which should be rare. And ES modules is the future, so maybe that's OK?

**Comment by LinusU** [comment]
> Only where they are mixed.

Do you mean that it won't touch `require`s, and only concern itself with the imports?

I don't have a strong opinion on which plugin we use, although we already include the `import` plugin. What are the pros of using `simple-import-sort` instead?

**Comment by mightyiam** [comment]
The simple import sort is a one-stop-shop with very little configuration. Probably none that we would use. If we like it — we like it. Nothing more to do. It's kind of similar to standard in this sense.

If the `import` plugin allows a similar functionality plus it supports `require`, then it may be preferable. Does it support `require`?
